dev/com.ibm.ws.security.social/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -238,3 +238,19 @@ discoveryPollingRate.desc=Rate in milliseconds at which the OpenID Connect clien
 
 forwardLoginParameter=Forward login parameter
 forwardLoginParameter.desc=Specifies a comma-separated list of parameter names to forward to the OpenID Connect provider. If a protected resource request includes one or more of the specified parameters, the OpenID Connect client will include those parameters and their values in the authorization endpoint request to the OpenID Connect provider.
+
+k8sTokenReviewEndpoint=Kubernetes Token Review Endpoint
+k8sTokenReviewEndpoint.desc=URL for the Kubernetes TokenReview API that is used to obtain information about an authenticated user from an access token.
+
+serviceAccountTokenForK8sTokenreview=Service account token for Kubernetes Token Review
+serviceAccountTokenForK8sTokenreview.desc=Token for OpenShift service account with the system:auth-delegator role. The token is required for making requests to the Kubernetes TokenReview API.
+
+useAccessTokenFromRequest=Use access token from request
+useAccessTokenFromRequest.desc=Determines if the access token that is provided in the request is used for authentication.
```

Comment:
Change
"Determines if the access token that is provided in the request is used for authentication."
To
"Determines **whether** the access token that is provided in the request is used for authentication."